modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectFieldRelationshipTypeException.java

```diff
@@ -38,4 +38,15 @@ public ObjectFieldRelationshipTypeException(Throwable throwable) {
 		super(throwable);
 	}
 
+	public static class MustNotDeleteObjectFieldRelationshipType
```

Comment:
hey @gabrielwas , 
Do you think that this is the right way to add a exception? I was wondering because Brian commented in Mateus's PR complaining about this and I think I did the same here: https://github.com/brianchandotcom/liferay-portal/pull/111771

Reply:
@marcelabc we should fix here what Brian mentioned there: Turning these public constructors into a private one.